test/lit/passes/unsubtyping-casts.wast

```diff
@@ -34,20 +34,19 @@
  ;; CHECK:      (rec
  ;; CHECK-NEXT:  (type $top (sub (struct)))
  (type $top (sub (struct)))
- ;; CHECK:       (type $mid (sub $top (struct)))
  (type $mid (sub $top (struct)))
- ;; CHECK:       (type $bot (sub $mid (struct)))
+ ;; CHECK:       (type $bot (sub $top (struct)))
```

Comment:
This no longer tests what it means to. I will fix it. Same with the other tests in this file.